src/cascadia/TerminalApp/AppLogic.cpp

```diff
@@ -616,9 +629,9 @@ namespace winrt::TerminalApp::implementation
         try
         {
             auto newSettings = _isUwp ? CascadiaSettings::LoadUniversal() : CascadiaSettings::LoadAll();
-            _settings.copy_from(winrt::get_self<CascadiaSettings>(newSettings));
+            _settings = newSettings;

```

Comment:
to save eight nanoseconds you could do uh

```suggestion
            _settings = std::move(newSettings);
```

or just `_settings = isUwp ? xxx : yyy`